ocp_install_env.sh

```diff
@@ -117,6 +117,12 @@ cat <<EOF
     provisioningBridge: ${PROVISIONING_NETWORK_NAME}
     provisioningNetworkCIDR: $PROVISIONING_NETWORK
     provisioningNetworkInterface: $CLUSTER_PRO_IF
+EOF
+  fi
+  # A predictable MAC is needed to configure a static IP
```

Comment:
I don't think this is needed, the interface is being specified by name, can re remove all the MAC generation?

Reply:
Ah good point, we probably can - unless we want to gain coverage for the `externalMACAddress` field?

Provided this works without I'll remove it, and perhaps propose another variable that allows the predictable MAC (and we can decide later if/where to enable that in CI)